components/raftstore-v2/src/operation/command/admin/conf_change.rs

```diff
@@ -0,0 +1,509 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+//! This module implements the configuration change command.
+//!
+//! The command will go through the following steps:
+//! - Propose conf change
+//! - Apply after conf change is committed
+//! - Update raft state using the result of conf change
+
+use collections::HashSet;
+use engine_traits::{KvEngine, RaftEngine};
+use kvproto::{
+    metapb::{self, PeerRole},
+    raft_cmdpb::{AdminRequest, AdminResponse, ChangePeerRequest, RaftCmdRequest},
+    raft_serverpb::{PeerState, RegionLocalState},
+};
+use protobuf::Message;
+use raft::prelude::*;
+use raft_proto::ConfChangeI;
+use raftstore::{
+    store::{
+        metrics::{PEER_ADMIN_CMD_COUNTER_VEC, PEER_PROPOSE_LOG_SIZE_HISTOGRAM},
+        util::{self, ChangePeerI, ConfChangeKind},
+        ProposalContext,
+    },
+    Error, Result,
+};
+use slog::{error, info, warn};
+use tikv_util::box_err;
+
+use super::AdminCmdResult;
+use crate::{
+    batch::StoreContext,
+    raft::{Apply, Peer},
+    router::ApplyRes,
+};
+
+/// The apply result of conf change.
+#[derive(Default, Debug)]
+pub struct ConfChange {
```

Comment:
It's easy to confuse with raft::eraftpb::ConfChange. 
How about renaming it to ConfChangeRes